rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -99,7 +125,8 @@ def test_add_edit_delete_entries(rotkehlchen_api_server: 'APIServer'):
         contained_in_msg='Failed to add event to the DB. It already exists',
         status_code=HTTPStatus.CONFLICT,
     )
-    # test editing works
+
+    # test editing evm event works
```

Comment:
Don't you have here below a copy-paste of what is being done in `assert_editing_works`?